Dockerfile

```diff
@@ -25,19 +25,14 @@ ARG VCS_REF="NULL"
 # Specify docker image metadata
 LABEL org.label-schema.schema-version="1.0"
 LABEL org.label-schema.name="carma-base"
-LABEL org.label-schema.description="Base operating system install for the CARMA Platform"
+LABEL org.label-schema.description="Base operating system install for the CARMA Platform on ARM architecture"
 LABEL org.label-schema.vendor="Leidos"
 LABEL org.label-schema.version=${VERSION}
 LABEL org.label-schema.url="https://highways.dot.gov/research/research-programs/operations/CARMA"
 LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-platform"
 LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
-# Specify which platform GPUs are available inside the container
-ENV NVIDIA_VISIBLE_DEVICES=${NVIDIA_VISIBLE_DEVICES:-all} \
-        # Specify which driver libraries/binaries will be mounted inside the container
-        NVIDIA_DRIVER_CAPABILITIES=${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
-
 # Avoid interactive prompts during the building of this docker image
 ARG DEBIAN_FRONTEND="noninteractive"
```

Comment:
I think this is supposed to be `ENV`. I don't think `ARG` affects the environment.

Reply:
removed since it was already set in the dustynv image